tests/test_populus_project.py

```diff
@@ -0,0 +1,18 @@
+from os.path import join, dirname, abspath
+
+import nkms_eth
+
+
+def test_testerchain_create(testerchain):
+    assert testerchain._network == 'tester'
+    assert testerchain._chain.web3.eth.blockNumber >= 0
+
+
+def test_nucypher_populus_project(testerchain):
+    project_dir = join(dirname(abspath(nkms_eth.__file__)), 'project')
+
+    assert testerchain._populus_config._project_dir == project_dir           # blockchain instance
```

Comment:
So, I'm looking to see that the `_populous_config` is properly set?  The comment, "blockchain instance", doesn't speak to me.

Reply:
I added some additional commentary.